services/navigation/builtin/builtin.go

```diff
@@ -68,6 +72,7 @@ func init() {
 type Config struct {
 	Store              navigation.StoreConfig `json:"store"`
 	BaseName           string                 `json:"base"`
+	MapTypeName        string                 `json:"map_type"`
```

Comment:
Should this be called `MapTypeName` or just `MapType`? It looks like the other config params with `Name` are dependencies of the service

Reply:
Good call out on the anti-pattern, I agree and will change it to map type